<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input_face</class>
 <widget class="QWidget" name="input_face">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>691</width>
     <height>708</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

#widget{
	border:3px solid blue

}</string>
   </property>
   <widget class="QLabel" name="cam_input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>703</x>
     <y>6</y>
     <width>371</width>
     <height>708</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_2{
	border:3px solid blue
}
</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="title">
     <string>脸部消息</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Fps:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Face in database：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>Face in current time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fps">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>**</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FaceInDatabase">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>**</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FaceInNow">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
     </property>
     <property name="text">
      <string>**</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>第一步：选择照片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_take">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>第二步：输入姓名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sureName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>第三步：保存脸部信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>保存脸部信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_output_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ouput_info</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_showNums">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
